staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -365,6 +374,21 @@ func (d *DiscoveryClient) SwaggerSchema(version schema.GroupVersion) (*swagger.A
 	return &schema, nil
 }
 
+// OpenAPISchema fetches the open api schema using a rest client and parses the json.
+// Warning: this is very expensive (~1.2s)
```

Comment:
This is because the OpenAPI spec is a massive json file that takes a long time to deserialize into binary form.  This will be cached between calls as a binary, which loads much faster (~40ms).